### PR TITLE
Add spec-url for spread syntax itself

### PR DESCRIPTION
**Pull request by Josh-Cena**
#### Summary

Fix https://github.com/mdn/browser-compat-data/issues/12701
Fix https://github.com/mdn/browser-compat-data/issues/14392

### CODE REVIEW

File: javascript/operators/spread.json [hunk 1]
```diff
@@ -5,6 +5,11 @@
         "__compat": {
           "description": "Spread syntax (...)",
           "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/Spread_syntax",
+          "spec_url": [
+            "https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#prod-SpreadElement",
+            "https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#prod-ArgumentList",
+            "https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#prod-PropertyDefinition",
```
Comment (queengooborg) [inline javascript/operators/spread.json:11]: Small syntax error; JSON doesn't like tailing commas!

```suggestion
            "https://tc39.es/ecma262/multipage/ecmascript-language-expressions.html#prod-PropertyDefinition"
```

Comment (Josh-Cena) [reply javascript/operators/spread.json:11]: Oh oops, thanks for catching

**Review by queengooborg — APPROVED** [review]
Thanks for the quick fixes!  This is LGTM!

**Comment by Josh-Cena** [comment]
@queengooborg Do you think it actually makes sense to have `__compat` for spread itself? We are kind of conflating three features into one in the first place.

**Comment by Josh-Cena** [comment]
@queengooborg Do you think it actually makes sense to have `__compat` for spread itself? We are kind of conflating three features into one in the first place.

**Comment by Josh-Cena** [comment]
@queengooborg Do you think it actually makes sense to have `__compat` for spread itself? We are kind of conflating three features into one in the first place.

**Comment by queengooborg** [comment]
I think so, since it helps maintain parity with the MDN pages and structural continuity with other BCD features.  We do have subfeatures for each meaning of the spread syntax, though!

**Comment by Josh-Cena** [comment]
I see

**Pull request merged**